scene/gui/tab_bar.cpp

```diff
@@ -639,6 +704,33 @@ int TabBar::get_hovered_tab() const {
 	return hover;
 }
 
+bool TabBar::select_tab_on_left() {
+	const int offset_end = get_input_rollover() ? get_tab_count() : (get_current_tab() + 1);
```

Comment:
```suggestion
	const int offset_end = input_rollover ? get_tab_count() : (get_current_tab() + 1);
```
No need to use getter (same below).